src/test/java/next/reflection/Junit4TestRunner.java

```diff
@@ -2,10 +2,34 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.lang.reflect.Method;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
 public class Junit4TestRunner {
     @Test
     public void run() throws Exception {
         Class<Junit4Test> clazz = Junit4Test.class;
         // TODO Junit4Test에서 @MyTest 애노테이션이 있는 메소드 실행
+
+        Set<Method> testMethods = new LinkedHashSet<>();
+
+        for (Method method : clazz.getDeclaredMethods()) {
+            if (isTestMethod(method)) {
+                testMethods.add(method);
+            }
+        }
+
+        if(testMethods.size() >= 1) {
```

Comment:
IDE 의 reformat code 기능을 통해서 습관적으로 코드 정렬 하는 것을 추천드립니다. :)
```suggestion
        if (testMethods.size() >= 1) {
```